README.md

```diff
@@ -1,53 +1,342 @@
 # angular-fontawesome
 
-Angular component for Font Awesome 5, built with 
-[Angular Librarian](https://github.com/gonzofish/angular-librarian) and 
-conforming to the 
-[Angular Package Format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/edit?usp=sharing). 
+[![npm](https://img.shields.io/npm/v/@fortawesome/angular-fontawesome.svg?style=flat-square)](https://www.npmjs.com/package/@fortawesome/angular-fontawesome)
 
-## Basic Usage
+> Font Awesome 5 Angular component using SVG with JS
 
-Add an icon to an Angular template:
+Built with [Angular Librarian] and conforming to the [Angular Package Format].
 
+[Angular Librarian]: https://github.com/gonzofish/angular-librarian
+[Angular Package Format]: https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/edit?usp=sharing
+
+Hey there! We're glad you're here...
+
+#### Upgrading Font Awesome?
+
+If you've used Font Awesome in the past (version 4 or older) there are some
+things that you should learn before you dive in.
+
+> https://fontawesome.com/how-to-use/upgrading-from-4
+
+#### Get started
+
+This package is for integrating with Angular (not AngularJS). If you aren't using Angular then it's
+not going to help you. Head over to our "Get Started" page for some guidance.
+
+> https://fontawesome.com/get-started
+
+#### Learn about our new SVG implementation
+
+This package, under the hood, uses SVG with JS and the `@fortawesome/fontawesome-svg-core` library. This implementation differs drastically from
+the web fonts implementation that was used in version 4 and older of Font Awesome. You might head over there to learn about how it works.
+
+> https://fontawesome.com/how-to-use/svg-with-js
+
+#### Going from an older pre-release version?
+
+See [UPGRADING.md](./UPGRADING.md).
+
+You might also be interested in the larger umbrella project [UPGRADING.md](https://github.com/FortAwesome/Font-Awesome/blob/master/UPGRADING.md)
+
+## Installation
+
+```
+$ yarn add @fortawesome/fontawesome-svg-core
```

Comment:
It may be a good idea to eliminate this step to simplify installation. This package is also included in the `dependencies` of the `@fortawesome/angular-fontawesome` package, so users may end up with two copies of if versions don't match.

Reply:
Ah, this needs to be peer dependency. Let me double-check that.

Reply:
Good catch @devoto13, that would cause big trouble for anyone who tries to use the library.

Reply:
I would actually expect this to be a normal dependency, so it's less packages for users to install. Do you think it is a common use case to have more than one package depending on `@fortawesome/fontawesome-svg-core` or why do you want to have it as peer dependency?

Reply:
We need it to be a peer dependency because of the library. If multiple copies of `fontawesome-svg-core` exist in `node_modules` you could end up calling `library.add()` on a completely separate instance of the library. Then when, for example, the angular-fontawesome component goes to look up icons it's working off of a different instance of the library.

We saw this early on with each of our other components. Directory looks a bit like this:

```
- node_modules
  - @fortawesome
    - fontawesome-svg-core # used if you import { library } in your app
    - angular-fontawesome
      - node_modules
        - @fortawesome
          fontawesome-svg-core # used by angular-fontawesome when that component uses icon()
```

Maybe there is a better way to solve this. Some other way to have a Singleton that even multiple installed versions of FA can use. But peer dependencies solved it rather nicely.

Reply:
I see what you're talking about. Because users need to import from `fontawesome-svg-core` directly to access the icons library. I agree it makes sense to have it this way for now.